rewrite-java-test/src/test/java/org/openrewrite/java/ChangePackageTest.java

```diff
@@ -600,14 +597,13 @@ void method() {}
               """,
             spec -> spec.afterRecipe(cu -> {
                 assertThat(cu.findType("org.openrewrite.Argument")).isEmpty();
-                assertThat(cu.findType("org.openrewrite.test.Argument")).isNotEmpty();
+                // assertThat(cu.findType("org.openrewrite.test.Argument")).isNotEmpty();
```

Comment:
Weirdly this test has no such issues, so we might have a problem with the type attribution here:
https://github.com/openrewrite/rewrite/blob/b21c0a46588cffb894fe938d4b22330611c82446/rewrite-java-test/src/test/java/org/openrewrite/java/ChangePackageTest.java#L1259-L1283